tests/unit/lang/eeprom/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# define the test executable
+add_executable(language_eeprom_tests
+../../../../src/lang/translator.cpp
+# ../../../../src/common/eeprom.cpp
```

Comment:
if the eeprom is not needed (not even in the future), remove it from the cmake file

Reply:
it's gone